tachyon/math/base/big_int.h

```diff
@@ -789,6 +789,41 @@ struct ALIGNAS(internal::LimbsAlignment(N)) BigInt {
     }
   }
 
+  // TODO(chokobole): This can be optimized since the element of vector occupies
+  // fixed 2 bits, we can save much space. e.g, in a worst case for 4 limbs(254
+  // bits), 254 * 2 / 8 = 63.4 < 8 * sizeof(uint64_t).
+  std::vector<int8_t> FindNAF() const {
```

Comment:
How about adding what NAF is abbreviate of and maybe a reference link?
We know, but some one first seeing this might not know I guess.